src/xdocs/config_design.xml

```diff
@@ -1479,34 +1631,44 @@ public class ImmutableClass
         <source>
 &lt;module name=&quot;VisibilityModifier&quot;&gt;
   &lt;property name=&quot;allowPublicImmutableFields&quot; value=&quot;true&quot;/&gt;
-  &lt;property name=&quot;immutableClassCanonicalNames&quot; value=&quot;
-  com.google.common.collect.ImmutableSet&quot;/&gt;
+  &lt;property name=&quot;immutableClassCanonicalNames&quot;
+        value=&quot;com.google.common.collect.ImmutableSet, java.lang.String&quot;/&gt;
```

Comment:
If you are wrapping `value`, keep it consistent with `+2` indentation. Please do it for all the `value`s in all the configs.

Reply:
Done

Reply:
Resolved.